public/app/features/explore/LiveLogs.tsx

```diff
@@ -171,7 +187,7 @@ class LiveLogs extends PureComponent<Props, State> {
           </button>
           <button onClick={this.props.stopLive} className={cx('btn btn-inverse', styles.button)}>
             <i className={'fa fa-stop'} />
```

Comment:
Should we change the icon here? I think as we changed the text from `Stop` -> `Exit live mode` the stop icon makes less sense.

Reply:
@jessover9000 I think we can use the `fa-times` here (the small `x`) instead as you originally proposed.